starknet_py/cairo/serialization/factory.py

```diff
@@ -102,6 +102,16 @@ def serializer_for_payload(payload: Dict[str, CairoType]) -> PayloadSerializer:
     )
 
 
+def serializer_for_event(event: Abi.Event) -> PayloadSerializer:
```

Comment:
Added this as using payload serialiser is not obvious

Reply:
Should this close #367 ?